README.md

```diff
@@ -119,3 +122,22 @@ It is possible to install the proprietary packages via OPKG by doing the followi
 > **Note**:
 > The read-write filesystem is (currently) not large enough to contain all the above packages. It is possible to resize
 > the UBI volumes. However, that must be done with great care.
+
+## Restoring the Official Image
+
+Unless the above instructions have been performed multiple times, reverting back to the official image is as simple as
+booting the previously used boot slot (from Linux):
+```bash
+fw_setenv update_url  # Re-enable automatic updates
+fw_setenv bootslot $(( 1 - $(fw_printenv -n bootslot) ))
+reboot
+```
+
+To install the official image through TFTP, fetch the files for the [latest release][1] from our server, e.g.
```

Comment:
The latest might not work (yet). Maybe dropping a hint to try 2nd latest release if so?

Or will we just push out archive releases faster in the future?

Reply:
Normally we push the tag when the release is out, no?

Reply:
So far, yes. But we might soon want to deploy OPKG (archive) feeds already when building, and when doing so, we make stuff available to 3rd-parties and therefore might want to push the FOSS stuff too. Of not for our own sake, then for legal reasons.